extensions/VisualEditor/VisualEditor.i18n.php

```diff
@@ -493,6 +493,11 @@
   'visualeditor-wikitext-warning-link' => 'mediawikiwiki:Special:MyLanguage/Help:Formatting',
   'visualeditor-wikitext-warning-title' => 'Wikitext markup detected',
   'visualeditor-dialog-media-insert-title' => 'Insert media',
+  'visualeditor-dialog-image-insert-title' => 'Insert an image',
+  'visualeditor-dialog-video-insert-title' => 'Insert a video',
+  'wikia-visualeditor-dialogbutton-wikiamediainsert-tooltip' => 'Media',
+  'wikia-visualeditor-dialogbutton-imageinsert-tooltip' => 'Movie',
```

Comment:
Is that really correct value? - "Movie" for wikia-visualeditor-dialogbutton-**image**insert-tooltip